exporting/tests/test_exporting_engine.c

```diff
@@ -796,7 +793,7 @@ static void test_create_main_rusage_chart(void **state)
     expect_string(rrdset_create_custom, id, "exporting_main_thread_cpu");
     expect_value(rrdset_create_custom, name, NULL);
     expect_string(rrdset_create_custom, family, "exporting");
-    expect_value(rrdset_create_custom, context, NULL);
+    expect_string(rrdset_create_custom, context, "exporting_cpu_usage");
     expect_string(rrdset_create_custom, units, "milliseconds/s");
```

Comment:
Testing for exact values looks to excessive to me. @amoss

Reply:
An interesting comment, why do you want me to look at it?

Reply:
Because we had a short test discussion during our last o3 😄